### PR TITLE
Use glib 2.71 for minimum v2_72 pkg-config versions

### PR DESCRIPTION
**Pull request by jf2048**
Per https://github.com/gtk-rs/gtk-rs-core/pull/577#issuecomment-1058912597

Not sure this makes sense, we don't use odd version numbers for any of the other features?

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
> Not sure this makes sense, we don't use odd version numbers for any of the other features?

At least in gtk4-rs, we use that a lot. I guess we didn't in the past because we were not updating the bindings that often and were waiting until there is a new release and for the GIR files to land on debian/ubuntu repositories

**Comment by jf2048** [comment]
Hmm, ok. New ubuntu LTS is coming next month so hopefully CI can be updated then.

**Comment by bilelmoussaoui** [comment]
> Hmm, ok. New ubuntu LTS is coming next month so hopefully CI can be updated then.


We no longer Ubuntu for getting the gir files and I plan to update the images we use to build things directly from upstream so we can test new APIs as soon as they land upstream and ensure things are working fine for the rust bindings

**Comment by jf2048** [comment]
Sorry not that, I mean this one: https://github.com/gtk-rs/gtk-rs-core/blob/caa47b15e3ded4ea1c583c8b3b4fbb936e571462/.github/workflows/CI.yml#L12

**Comment by bilelmoussaoui** [comment]
Ah sorry misunderstood, but nonetheless we use an image built from https://github.com/gtk-rs/gtk-rs-core/blob/master/Dockerfile. I will update it this weekend

**Comment by sdroege** [comment]
> At least in gtk4-rs, we use that a lot.

In GStreamer I also do that since a long time, e.g. `v1_20` required 1.19.x to be installed until the 1.20.0 release.

**Review by sdroege — APPROVED** [review]

**Comment by sdroege** [comment]
The same also applies to pango btw, but we can worry about that once someone needs it.

**Pull request merged**
